indigo_app/views/tasks.py

```diff
@@ -100,9 +100,29 @@ def get_context_data(self, **kwargs):
         # merge actions and comments
         actions = task.action_object_actions.all()
         task_content_type = ContentType.objects.get_for_model(self.model)
-        comments = Comment.objects\
+        comments = list(Comment.objects\
             .filter(content_type=task_content_type, object_pk=task.id)\
-            .select_related('user')
+            .select_related('user'))
+        
+        # get the annotation for the particular task
+        task_annotation = Annotation.objects.filter(task=task).first()
+
+        # for the annotation that is linked to the task, get all the replies
+        if task_annotation:
+            fake_comments = []
+            # get the replies to the annotation
+            annotation_replies = Annotation.objects.filter(in_reply_to=task_annotation)
+            for annotation in annotation_replies:
```

Comment:
There's a simpler way to do this. You're building up a list one by one, then appending that list to another list. Instead, you can just append the items one by one to the other list. Even more pythonic, is this:

```python
comments.extend([Comment(
    user=a.created_by_user,
    comment=a.text,
    # etc.
) for a in annotation_replies])
```

Reply:
I usually go for readability first 😅but yes, that's a more Pythonic way of doing it.